.travis.yml

```diff
@@ -32,19 +32,21 @@ install:
   - pipenv install --dev
 
 # perform testing:
-# --> run the test suite with pytest
+# --> run the test suite with pytest and codecov
+# --> Check for black format
 # --> lint the functions with flake8
 # --> lint the test suites with flake8
 # --> lint the functions with pylint
 # --> lint the test suites with pylint
 # --> lint the README documentation with mdl
+# --> Run django server
```

Comment:
The script doesn't actually run the Django server. You should also look into [Django testing](https://docs.djangoproject.com/en/2.1/topics/testing/overview/) and deployment testing, probably in a different PR.

Reply:
I removed the actual script because it caused problems. I forgot to remove the comment